_docker/drupal/drupal-filesystem/web/themes/custom/rhdp/rhd-frontend/src/styles/assembly/_dynamic_content_list.scss

```diff
@@ -54,7 +71,7 @@
 
   .disqus {
     background: #FFF;
-    padding: 35px 1rem;
+    padding: 35px 20px;
```

Comment:
Why are we going back to fixed pixels?

Reply:
I can switch it to rems, if that's what you prefer, but the spacing was definitely not correct.

Reply:
Yeah, let's use rems.